project/export_presets.cfg.template

```diff
@@ -220,7 +220,7 @@ custom_features=""
 export_filter="all_resources"
 include_filter="*.json"
 exclude_filter=""
-export_path="export/html5/frog-finder.html"
+export_path="export/html5/index.html"
```

Comment:
Document this in the commit comment.

Reply:
Documented, `I changed the html export filename to 'index.html' so we no longer...`